### PR TITLE
3.0 updates

### PR DESCRIPTION
**Pull request by bicknellr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-Using svg elements to create icons has a few advantages over traditional
-bitmap graphics like jpg or png. Icons that use svg are vector based so
-they are resolution independent and should look good on any device. They
-are stylable via css. Icons can be themed, colorized, and even animated.
+Using svg elements to create icons has a few advantages over traditional bitmap
+graphics like jpg or png. Icons that use svg are vector based so they are
+resolution independent and should look good on any device. They are stylable
+via css. Icons can be themed, colorized, and even animated.
 
 Example:
```
Comment (notwaldorf) [inline README.md]: These are just copy pasted from the element docs, no? I think we're just linking to them, and deleting everything else, as per https://raw.githubusercontent.com/PolymerElements/paper-input/85c6cfd45ba0f0c6c76bceee93b5f18890e2c655/README.md

File: demo/index.html [hunk 2]
```diff
@@ -12,45 +12,32 @@
...
+  <script type="module">
+    import '@polymer/iron-demo-helpers/demo-snippet.js';
+    import '@polymer/iron-demo-helpers/demo-pages-shared-styles.js';
+    import '@polymer/iron-icon/iron-icon.js';
+    import '../iron-iconset-svg.js';
+    import './svg-sample-icons.js';
 
-document.body.appendChild($_documentContainer.content);
-</script>
+    const $_documentContainer = document.createElement('template');
```
Comment (notwaldorf) [inline demo/index.html]: we're using the `html` function for this now:
```
import {html} from '@polymer/polymer/lib/utils/html-tag.js';
const template = html``;
document.body.appendChild(template.content);
```

File: demo/svg-sample-icons.js [hunk 3]
```diff
@@ -1,8 +1,21 @@
...
+*/
+
 import '../iron-iconset-svg.js';
+
 const $_documentContainer = document.createElement('template');
 $_documentContainer.setAttribute('style', 'display: none;');
 
-$_documentContainer.innerHTML = `<iron-iconset-svg name="svg-sample-icons" size="100">
+$_documentContainer.innerHTML =
+    `<iron-iconset-svg name="svg-sample-icons" size="100">
```
Comment (notwaldorf) [inline demo/svg-sample-icons.js]: `html` function here too

File: README.md (hunk 1 above)
Comment (bicknellr) [reply README.md]: SGTM; I replaced this top section with the links in the example there.

File: demo/index.html (hunk 2 above)
Comment (bicknellr) [reply demo/index.html]: Fixed.

File: demo/svg-sample-icons.js (hunk 3 above)
Comment (bicknellr) [reply demo/svg-sample-icons.js]: This was the one you were mentioning offline that probably shouldn't use `html`, right? (left unchanged)

Comment (bicknellr) [reply demo/svg-sample-icons.js]: Or, maybe you were talking about [this comment](https://github.com/PolymerElements/iron-icons/pull/88#discussion_r203135953) instead?

**Pull request merged**